### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by mikethreels**
### when creating the project the following points have been added.
- [x] create the array to store the board.
- [x] create render function to show the array and additional information on the browser.
- [x] create a player factory.
- [x] created functions to check for a winner or a draw, change the visibility of sections, reset the game and initiate the game.

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @mikethreels 
### You have done well so far :clap: However, you have a few issues to fix:-
- [x] Organize your codebase in a nice directory e.g tic_tac_toe_js/src/ files inside
- [x] Separate the app logic from the dom manipulation, good practice
- [x] Remove the comments from the codebase
### Make the changes and submit for another code review. Good luck.
- Happy coding!

### [Optional]
- You can use localStorage to store the scores
- Improve the design interface of the app especially the form inputs

By Raymond
Slack @iamraymond

File: game.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+    messageContainer.append(messageText);
+    const buttonDiv = document.createElement('div');
+    buttonDiv.innerHTML = '<button id="reset_button">Play again!</button>';
+    message.append(messageContainer);
+    message.append(buttonDiv);
+    message.attributes.class.value = 'message_section';
+    game.attributes.class.value = 'none';
+    game_info.attributes.class.value = 'none';
+    document.querySelector('button').addEventListener('click', reset);
+  };
```
Comment (rOluochKe) [inline game.js]: ### Separate app dom manipulation from app logic

Comment (mikethreels) [reply game.js]: Hi Raymond,

Thank you very much for the great feedback!

have a great day,
Michael.

**Comment by mikethreels** [comment]
## Changelog: tic-tac-toe

### The below changes have been made to the project.
- [x] Added a directory to house the codebase.
- [x] Deleted comments from the code.
- [x] Separated logic from the dom manipulation.

**Review by bellom — APPROVED** [review]
## Hi @mikethreels  :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next project :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**